src/fetch_family_allowance.py

```diff
@@ -0,0 +1,91 @@
+import datetime
+import warnings
+import requests
+import zipfile
+from urllib.request import urlretrieve
+import os
+import pandas
+import fileinput
+from sqlalchemy import create_engine
+import sys
+import shutil
+
+STARTING_YEAR = 2011
+
+NOW = datetime.datetime.now()
+
+SERVER = 'http://arquivos.portaldatransparencia.gov.br/'
+PATH = 'downloads.asp?a={}&m={:02d}&consulta=BolsaFamiliaFolhaPagamento'
+outputfile = 'data_family_allowance.csv'
+BASE_DATA_DIR = '../data/'
+TEMP_DATA = "../data/temp_data/"
```

Comment:
Can we use [`tempfile.TemporaryDirectory`](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory)?